python/langsmith/evaluation/_runner.py

```diff
@@ -452,6 +452,36 @@ def _process_data(self) -> None:
     def __len__(self) -> int:
         return len(self._results)
 
+    def _repr_html_(self) -> str:
+        import importlib.util
+
+        if self._results and importlib.util.find_spec("pandas"):
```

Comment:
Can we make a `to_pandas` method and then re-use that here?

Would be useful in general